qase-javascript-commons/README.md

```diff
@@ -1,102 +1,79 @@
-| qaseCoreReporterOptions.`screenshotFolder`  | `string`  | no       | Folder to find screenshots with Qase ID in file name                              |
-| qaseCoreReporterOptions.`videoFolder`       | `string`  | no       | Folder to find videos with Qase ID in file name                                   |
-| qaseCoreReporterOptions.`loadConfig`        | `boolean` | no       | Whether or not to also load reporter options from `qase.config.json` or `.qaserc` |
-
-## API (public methods)
-
-<a name="qase-core-reporter-public-methods"></a>
-
-| Method                                   | Description                                                                                                     |
-| ---------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
-| start()                                  | Check if reporting is enabled, check project and create run if needed                                           |
-| addTestResult(test, status, attachments) | Parse and save test results for publishing                                                                      |
-| end({spawn: true/false})                 | Upload attachments if needed and map to existing results, send results as BulkCreate, complete run if specified |
-| uploadAttachments(attachments)           | Upload attachments based on file path and return attachment hashes for test results                             |
-
-## Qase Reporter Environmental Variables
-
-> Qase environmental variables are first class options, which means the user can change all default/static reporter options by using these variables.
-
-<a name="qase-core-reporter-envs"></a>
-| Variable | Type | Description |
-| ------------------------- | --------- | -------------------------------------------------------------- |
-| `QASE_REPORT` | boolean | Whether or not to enable reporter |
-| `QASE_API_TOKEN` | string | Qase api key |
-| `QASE_API_BASE_URL` | string | Qase API base url |
-| `QASE_PROJECT_CODE` | string | Qase project ID |
-| `QASE_RUN_ID` | string | Qase run ID, if you want results to be posted to a specific run |
-| `QASE_RUN_NAME` | string | Qase run custom name |
-| `QASE_RUN_DESCRIPTION` | string | Qase run custom description |
-| `QASE_RUN_COMPLETE` | string | Whether or not to complete a test run when results are posted |
-| `QASE_ENVIRONMENT_ID` | string | Qase environment ID |
-| `QASE_ROOT_SUITE_TITLE` | string | The root suite that unknown test cases will be added to |
-| `QASE_UPLOAD_ATTACHMENTS` | string | Whether or not to upload attachments
-| `QASE_LOGGING` | boolean | Whether or not to output Qase logs while testing and reporting |
+## Configuration
+
+Qase JS Reporters can be configured in multiple ways:
+
+- using a config file `qase.config.json`
+- using environment variables
+
+All configuration options are listed in the table below:
+
```

Comment:
I hope you know about https://www.tablesgenerator.com/markdown_tables

Reply:
It makes editing markdown tables much easier

Reply:
I didn't know about this site. Thanks!